package.json

```
@@ -342,7 +342,7 @@
 		"stylelint": "^9.10.1",
 		"supertest": "^4.0.2",
 		"svgstore-cli": "^1.3.1",
-		"terser": "^4.6.11",
+		"terser": "4.6.13",
```

You shouldn't ping this to a specific version. You can update the version number while keeping this a version range:

```suggestion
		"terser": "^4.6.13",
```